packages/components/src/internal/components/editable/Cell.tsx

```diff
@@ -15,23 +15,24 @@
  */
 import React from 'react';
 import classNames from 'classnames';
-import { List } from 'immutable';
-import { OverlayTrigger, Popover } from 'react-bootstrap';
-import { Query } from '@labkey/api';
+import {List} from 'immutable';
+import {OverlayTrigger, Popover} from 'react-bootstrap';
+import {Query} from '@labkey/api';
 
-import { cancelEvent, isCopy, isPaste, isSelectAll } from '../../events';
-import { CellMessage, ValueDescriptor } from '../../models';
-import { KEYS, MODIFICATION_TYPES, SELECTION_TYPES } from '../../constants';
+import {cancelEvent, isCopy, isFillDown, isPaste, isSelectAll} from '../../events';
+import {CellMessage, ValueDescriptor} from '../../models';
+import {CELL_SELECTION_HANDLE_CLASSNAME, KEYS, MODIFICATION_TYPES, SELECTION_TYPES} from '../../constants';
 
-import { QueryColumn } from '../../..';
+import {QueryColumn} from '../../..';
 
-import { getQueryColumnRenderers } from '../../global';
+import {getQueryColumnRenderers} from '../../global';
 
-import { LookupCell, LookupCellProps } from './LookupCell';
-import { DateInputCell, DateInputCellProps } from './DateInputCell';
+import {LookupCell, LookupCellProps} from './LookupCell';
+import {DateInputCell, DateInputCellProps} from './DateInputCell';
```

Comment:
not sure why the spaces on these imports got updated...I'll lint before merging.